packages/apify-extra/package.json

```diff
@@ -60,5 +60,8 @@
     "peerDependencies": {
         "apify": ">= 3.0.0",
         "crawlee": ">= 3.0.0"
+    },
+    "devDependencies": {
+        "apify": "^3.0.0"
```

Comment:
i doubt its about this, the package is locally defined and mapped via tsconfig, this shouldnt be really used locally as paths mapping should have preference.

Reply:
imo it's a "race condition" coming from `turbo`  - without the declared dependency, `turbo` builds `apify` and `apify-extra` in parallel (`rimraf`? I'm a bit puzzled why would it look into the `dist` folder, but I should just read up on this more).

Anyway, if you prebuild `apify` with 
`npx turbo run build --filter=apify `
and then go for `npm run build` (with the `apify` build cached), the build finishes w/o problems.

Reply:
thats not really a solution, workaround at best. turbo should respect peer dependencies and resolve things correctly, thats what its for. if you think its caused by turbo doing things in parallel, try disabling the concurrency, but i doubt it.

Reply:
It is caused by turborepo: 
https://github.com/vercel/turborepo/issues/1601

Rebased this branch here, specifying this dependency in `turbo.json`:
https://github.com/apify/apify-sdk-js/pull/108

Reply:
i can see it fails the same with lerna without the explicit dev dep, weird but tolerable. weird, iirc i was trying this out already. there are other build failures but those are probably caused by the PR not being rebased

Reply:
lets use the dev dependency, better than adjusting turbo config i guess, i thought it wasnt enough